collect_app/src/main/java/org/odk/collect/android/projects/QrCodeProjectCreatorDialog.kt

```diff
@@ -88,6 +125,26 @@ class QrCodeProjectCreatorDialog : MaterialFullScreenDialogFragment(), Duplicate
         return binding.root
     }
 
+    private fun configureMenu() {
+        binding.toolbar.menu.removeItem(R.id.menu_item_share)
+
+        binding.toolbar.setOnMenuItemClickListener {
+            when (it.itemId) {
+                R.id.menu_item_scan_sd_card -> {
```

Comment:
This the only option in the tootlbar. It would be better to have an icon for it instead of hiding it. It might be a separate issue since if we decide to add such and icon we need to do the same for 'Reconfigure with qr code' screen.

Reply:
Not displaying it is intentional. When we previously considered it we wanted to make sure it wasn’t eye-catching and that we had an opportunity to use text to explain what it does. It still feels like the right decision to me.

Reply:
Sorry, was going quickly so I think that came across as more aggressive than I intended! Fundamentally, you're right that displaying a single item in an overflow menu is an anti-pattern. I think here it's kind of the "least bad" alternative. Most importantly, it matches the prior workflow so meets our most important goal of helping users who already have a process built around this functionality.